nss-ivory-trolls/BangazonSite
# Cb homepage

# Description

The home page now displays the 20 most recently created products and links to each of the product's details page. Also, the create products view now validates for restricting special characters in the description and title inputs and limits the price to under $10,000.

## Type of change


# Testing Instructions for Change Made

1. `git fetch -a`
1. `git checkout cb-homepage`
1. Please verify that the homepage displays correctly as described above.
1. Please verify that the product create view validates the inputs as described above.

# Checklist:
